src/odemis/acq/test/move_test.py

```diff
@@ -353,6 +412,14 @@ def test_moving_to_grid1_in_fm_imaging_area_after_loading(self):
         grid_label = self.posture_manager.getCurrentGridLabel()
         self.assertEqual(position_label, FM_IMAGING)
         self.assertEqual(grid_label, GRID_1)
+        # check the values of tilt and rotation
+        fm_angles = self.stage.getMetadata()[model.MD_FAV_FM_POS_ACTIVE]
+        if self.stage_version == "zeiss_1":
+            self.assertAlmostEqual(self.stage.position.value["rx"], fm_angles["rx"], places=5)
+            self.assertAlmostEqual(self.stage.position.value["rm"], fm_angles["rm"], places=5)
+        else:
+            self.assertAlmostEqual(self.stage.position.value["rx"], fm_angles["rx"], places=5)
+            self.assertAlmostEqual(self.stage.position.value["rz"], fm_angles["rz"], places=4)
```

Comment:
Relevant rotational axes values in yaml file has 4 decimal places. In simulation the stage moves to the specified decimal places as in yaml file and the higher decimal places are given random non-zero values, I think. 

Now I also see the test cases for linear axes movement supporting until 4 decimal places.  

To tackle it, need to maybe modify the functions in tmcm.py in driver/ like _doMoveRel, etc. Is that correct @pieleric ?

Reply:
Rx is also a rotational axis right? Why is that line also 5 places?

Reply:
I have changed the code to check upto 4 places. It looks sufficient to check upto 4 places. Now it is uniform.

Reply:
It would be nice if you could provide a small description how you came to the conclusion that it looks sufficient to check up to 4 places. I mean for the  rotational axis I completely get that as 5 is already measuring in a tenth of millidegrees in which the difference might not even be measured. But for linear axis which are measured in m 4 is measuring in a tenth of a mm which is in relative terms quite a lot right?

Reply:
It is checking the rotational axes with 4 decimal places( you have provided good explanation). It is not checking directly the linear axes by using assert, hence there is no decimal check for linear axes. The linear axes movement is checked indirectly by checking if the linear position is in desired imaging mode (SEM, FM, LOADING) or not.

Reply:
All the axes with prefix r, are rotational axes.

Reply:
I see and I think I got a little dragged onto this comment of yours, assuming there were also assert checks for positions for the linear axes:

> Now I also see the test cases for linear axes movement supporting until 4 decimal places.

So I guess for now its fine like it is done now. Thanks for you explanation.